src/Shared/E2ETesting/E2ETesting.targets

```diff
@@ -7,7 +7,7 @@
   <Target Name="EnsureNodeJSRestored" Condition="'$(SeleniumE2ETestsSupported)' == 'true'" BeforeTargets="Build">
     <Message Text="Running yarn install on $(MSBuildProjectFile)" Importance="High" />
 
-    <Message Condition="'$(EnforcePrerequisites)' == ''"
+    <Message Condition="'$(EnforceE2ETestPrerequisites)' == ''"
```

Comment:
` '$(EnforceE2ETestPrerequisites)' != 'true' `

Reply:
I'm not going to be changing '== ''' to '!= 'true'' this is a well stablished msbuild convention and is not the focus of this PR.

Reply:
The supposed convention you mention is _wrong_ and isn't used in most of our infrastructure. The problem is it prevents use of the explicit `/p:EnforceE2ETestPrerequisites=false` on the command line: An empty property means use the default, not `false`.

Reply:
I agree. We mostly use `'$(EnforceE2ETestPrerequisites)' != 'true'` convention which respects command line options.

Reply:
Ok, updated.